### ISSUE TITLE
Fehlermeldung beim Anlegen eines Nutzerkontos anpassen

### ISSUE THREAD

**Issue by sigwei** [OPUS4/application#486]
Wenn versucht wird, ein Nutzerkonto mit einem Benutzernamen anzulegen, der einen Großbuchstaben enthält, so läuft das auf einen Fehler. Die Fehlermeldung ist aber irreführend, da nicht auf die Schreibweise mit Kleinbuchstaben hingewiesen wird. Die Fehlermeldung soll daher angepasst werden.

Übersetzungsschlüssel: _validation_error_login_regexNotMatch_

alt:
Benutzernamen dürfen **Buchstaben**, Zahlen und folgende Zeichen enthalten, '@._-'.
Login can contain **letters**, numbers and the following characters, '@._-'.

neu:
Benutzernamen dürfen **Kleinbuchstaben**, Zahlen und folgende Zeichen enthalten, '@._-'.
Login can contain **lowercase letters**, numbers and the following characters, '@._-'.

**Comment by j3nsch** [comment]
Auf den ersten Blick könnte man auch die Einschränkung lockern. Ich vermute sie existiert, damit nicht zwei Accounts existieren können, die sich nur durch die Groß- und Kleinschreibung unterscheiden. Es macht vermutlich Sinn Account-Namen case-insensitive zu behandeln. Dann bräuchte es diese Einschränkung nicht mehr, dann wäre es egal.

**Comment by sigwei** [comment]
Ja, Account-Namen "case-insensitive" zu behandeln wäre die schönere Lösung. Bei den Nutzerrollen scheint das bereits so gelöst zu sein.
